src/to_be_raptor_linx

```diff
@@ -10,6 +10,8 @@ RAPTOR_PATH=`( cd "$SCRIPT_PATH" && pwd )`
 
 [ -f $RAPTOR_PATH/../.raptorenv_lin64.sh ] && source $RAPTOR_PATH/../.raptorenv_lin64.sh
 
+export LICENSE_LOCATION=$HOME/.local/Rapidsilicon/raptor.lic\;$LICENSE_LOCATION
```

Comment:
is that possible to take this path from etc/config.json? otherwise we have copy/paste code in very different places and this will be hard to handle in future